examples/boxed_values/models/pggen.toml

```
@@ -0,0 +1,12 @@
+[[table]]
+    name = "users"
+
+[[query]]
+    name = "GetUsersFromGmail"
+    not_null_fields = ["nickname"]
+    # The query to run against the database. For longer queries multi-line strings using
+    # ''' are often useful.
+    body = '''
+    SELECT * FROM users where email like '%gmail.com'
```

I've been trying to follow the style where sql keywords are in ALL_CAPS. Since this example is both documentation and a test, can you follow that style.
```suggestion
    SELECT * FROM users WERE email LIKE '%gmail.com'
```

---

You typo'd where but I got it changed

---

Oh whoops, thanks